rest/slurmrestapi.py

```diff
@@ -503,5 +546,22 @@ def proxy():
                            url_root=request.url_root,
                            masters=masters)
 
+
+def check_private_data_for_entity(user, entity):
+    onlyUsersEntities = False
```

Comment:
The code sets `currentUser` with `user` but `currentUser` then receives the result of `get_current_user()` before being actually used if the if check. If you remove `currentUser = user` the function is the same. Then, no, the argument is definitely not used.